dagshub/data_engine/annotation/voxel_conversion.py

```diff
@@ -11,6 +11,13 @@
 
 
 def add_voxel_annotations(sample: "fo.Sample", datapoint: "Datapoint", *annotation_fields: str):
+    """
+    Adds annotation to the voxel sample.
```

Comment:
Internal-ish function. Adds annotations to the sample from the datapoint that were stored in the *annotation_fields. Assumes the annotations are JSON blobs of voxel annotations.
It's called by the visualize() function if the voxel_annotations arg is specified.

Reply:
How can the user know what format needs to be provided for this? Is there a link in the Voxel docs?

Reply:
The format is 
If you want this documented then this should be a "workflow" type of doc on the regular docs talking about how to add annotations, and there should be a section "I have label studio annotations/I want to work on dagshub", and another one for "I have voxel annotations already". This whole thing was basically built for a client usecase that had their annotations already in voxel
This function isn't supposed to be called by users directly, but it might be useful for someone who's trying to build out their own weird workflow.